lib/scss/theme.scss

```diff
@@ -0,0 +1,36 @@
+// Theme Colors
+$text-muted-color: #606060;
+$border-color: #f0f0f0;
+
+
+// Font
+$font-size-base: 18px;
+$font-family: 'Source Sans Pro', sans-serif;
```

Comment:
Don't we also need to load `Source Sans Pro` in the HTML file? Currently I am seeing the font as `sans-serif`. @deekoder told me that we would have to link the font on the index.erb file of minio/gluegun similar to [this](https://github.com/miniohq/gluegun/pull/7/files#diff-55eeb0b31f5e803387ff502806629027R17) and control it with a variable.